pkg/plugins/policies/core/matchers/dataplane.go

```diff
@@ -129,6 +129,9 @@ func dppSelectedByPolicy(
 ) ([]core_rules.InboundListener, []core_rules.InboundListenerHostname, bool, error) {
 	switch ref.Kind {
 	case common_api.Mesh:
+		if !ddpSelectedByNamespace(meta, dpp) {
```

Comment:
it's not just for kind Mesh, it should be checked for all kinds no?

Reply:
you are right it applies to all types, changed